app/views/OfferedCourses/CoursesList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, View, FlatList, StyleSheet} from 'react-native';
+import CourseItem from "./CourseItem";
+
+
+
+class CoursesList extends React.Component {
+
+    static propTypes = {
+        courses: PropTypes.array.isRequired,
+    };
+
+    render() {
+        return (
+            <View style={styles.list}>
+               {/* {this.props.courses.map((course) =>
+                <Text key={course.key}> {course.name.en}
+                </Text>
+                )}*/}
+
+                <FlatList
+                    data={this.props.courses}
+                    renderItem={({item}) => <CourseItem course={item}/>}
```

Comment:
I think you have to somehow loop the items here, to print out the desired list. With .map for example.

What comes to mind is:
courses.map((course) =>
<CourseItem course={course} />
);

Reply:
There is no problem with functionality, commented code is just an old version, I’ll do a code cleanup when the functionality will be completed

Reply:
I would strongly recommend to clean this code before merging, it will be living hell trying to figure out what files need to be cleaned if four developers merge dirty code to master.

Reply:
Sure, I don’t get the problem at this stage since we don’t have that much of the code yet plus I will still work on this component. Unnecessary comments will be deleted upon completing the functionality anyway